pkg/volume/flocker/flocker_util.go

```diff
@@ -68,7 +67,6 @@ func (util *FlockerUtil) CreateVolume(c *flockerVolumeProvisioner) (datasetUUID
 	}
 
 	// select random node
-	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
...wat...

Reply:
i believe that file uses the api-machinery rand, so this line should be able to be safely removed.

Reply:
I agree it is safe to remove. I'm more confused how it existed in the first place. We are reseeding a global rand every call to this function. I'm surprised this wasn't more problematic.

Reply:
I think I saw two more weird places like this, but they were in vendor. One in aws-go-sdk and another one in some azure thing IIRC.